code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -34,94 +34,501 @@
 	if(Uses)
 		grind_results["slimejelly"] = 20
 
+//Effect when activated by a Luminescent. Separated into a minor and major effect. Returns cooldown in deciseconds.
+/obj/item/slime_extract/proc/activate(mob/living/carbon/human/user, datum/species/jelly/luminescent/species, activation_type)
+	to_chat(user, "<span class='notice'>Nothing happened... This slime extract cannot be activated this way.</span>")
+	return 0
+
 /obj/item/slime_extract/grey
 	name = "grey slime extract"
 	icon_state = "grey slime extract"
 
+/obj/item/slime_extract/grey/activate(mob/living/carbon/human/user, datum/species/jelly/luminescent/species, activation_type)
+	switch(activation_type)
+		if(SLIME_ACTIVATE_MINOR)
+			var/obj/item/reagent_containers/food/snacks/monkeycube/M = new
+			if(!user.put_in_active_hand(M))
+				M.forceMove(user.drop_location())
+			playsound(user, 'sound/effects/splat.ogg', 50, 1)
+			to_chat(user, "<span class='notice'>You spit out a monkey cube.</span>")
+			return 120
+		if(SLIME_ACTIVATE_MAJOR)
+			to_chat(user, "<span class='notice'>Your [name] starts pulsing...</span>")
+			if(do_after(user, 40, target = user))
+				var/mob/living/simple_animal/slime/S = new(get_turf(user), "grey")
```

Comment:
Slimes are dangerous when released, as are monkeys in large enough groups with monkey AI, it seems over convenient for a xeno-biologist instead to take this luminary over appropriately managing their stocks to save some up grey cores in reserve. A hugbox '**can never lose**' condition rather than QoL.

Reply:
Grey cores are literally the easiest to get, and when injected with blood they give three cores instead of one. Each. Luminescents won't top that.
As for slimes, this takes 4 seconds of standing still, instead of being instant.